iOS/Detail/DetailViewController.swift

```diff
@@ -168,6 +168,8 @@ class DetailViewController: UIViewController {
 		}
 		let itemSource = ArticleActivityItemSource(url: url, subject: navState?.currentArticle?.title)
 		let activityViewController = UIActivityViewController(activityItems: [itemSource], applicationActivities: nil)
+		activityViewController.popoverPresentationController?.barButtonItem = self.actionBarButtonItem
```

Comment:
This is the crash fix. The rest of the PR is related to add account presentation stuff.